canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-select-field-function.jsx

```diff
@@ -638,7 +644,8 @@ export default class ExpressionSelectFieldOrFunction extends React.Component {
 				<Layer level={this.props.controller.getLight() ? 1 : 0}>
 					<Dropdown
 						id={"properties-expression-field-select-dropdown-" + this.uuid}
-						label={label}
+						label="Choose from below Field List ..."
```

Comment:
This would need to be translated.

Reply:
Do we need to add label ? I have added this just because it was throwing below error in console.
<img width="460" alt="Screenshot 2024-06-12 at 5 19 42 PM" src="https://github.com/elyra-ai/canvas/assets/16302714/6dbbe461-488a-4fcf-8a81-a003ebd3a88b">

Reply:
Yes, it's required.

Reply:
This above label text is not displayed because of selectedItem so directly using same selectedItem text in label, please reivew.